src/components/Heroes/columns.jsx

```diff
@@ -23,72 +22,72 @@ export default {
     displayName: strings.hero_pick_ban_rate,
     field: 'pickBanRatePro',
     sortFn: true,
-    displayFn: (row, col, field) => percentileDisplay(field, row.matchCountPro),
+    percentBars: row => decimalToCount(row.pickBanRatePro, row.matchCountPro),
```

Comment:
Isn't it: 
* relativeBars takes a count value and divides by the max in the column
* percentBars takes a decimal value (between 0 and 10 and divides by 1 always